osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyCursorParticles.cs

```diff
@@ -83,7 +83,7 @@ protected override void Update()
             DrawableHitObject kiaiHitObject = null;
 
             // Check whether currently in a kiai section first. This is only done as an optimisation to avoid enumerating AliveObjects when not necessary.
-            if (gameplayBeatmap.ControlPointInfo.EffectPointAt(gameplayBeatmap.Time.Current).KiaiMode)
+            if (gameplayBeatmap.ControlPointInfo.EffectPointAt(Time.Current).KiaiMode)
```

Comment:
This was supposed to be using `gameplayClock` rather than `gameplayBeatmap` (see `Resolved` DI prop). If that's not needed then it can be removed.

Reply:
I see, removed.